sashimi-webapp/src/components/file-manager/Folder.vue

```diff
@@ -35,15 +35,10 @@
       saveFolderName() {
         window.getSelection().removeAllRanges();
 
-        let newFolderName = this.$el.getElementsByTagName('p')[0].innerHTML;
+        let newFolderName = this.$el.getElementsByTagName('p')[0].innerText;
```

Comment:
Instead of using `getElementsByTagName` it will be cleaner and better to use Vue's `refs` attribute instead.
Such that:
```javascript
let newFolderName = this.$refs.nameField.innerText;
```